src/transformers/agents/llm_engine.py

```diff
@@ -75,8 +75,6 @@ def __call__(self, messages: List[Dict[str, str]], stop_sequences=[]) -> str:
         if "Meta-Llama-3" in self.model:
```

Comment:
Why only hard code llama3, shouldn't we add the stop token according to model's tokenizer config?

Reply:
This was because of a specific Meta-Llamaa-3 issue in our inference API, since this model is one of the rare ones with more than one stop tokens!
For my own reference, [here](https://huggingface.slack.com/archives/C05CFK1HM0T/p1713804848669749) is a link to our internal slack message about it.

But it seems this problem has been solved [here](https://huggingface.co/meta-llama/Meta-Llama-3-70B-Instruct/commit/6100a3f3f907fd2bb80ed0e79c1f170ade0bc6ce). So normally we can remove it.